src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$color: #00acdc;
+$main-color: #00acdc;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+$color-darkly-blue: #34568b;
+$back-color: #fff;
```

Comment:
The variable name '$back-color' is unclear, consider renaming it to something more descriptive, like '$background-color'.